src/python/pants/option/global_options.py

```diff
@@ -65,6 +65,13 @@ def register_bootstrap_options(cls, register):
                       'pants.backend.python',
                       'pants.backend.jvm',
                       'pants.backend.codegen',
```

Comment:
Will that make warnings like
https://github.com/pantsbuild/pants/blob/master/src/python/pants/backend/codegen/targets/java_thrift_library.py#L12
become errors now?